frontend/src/pages/agreements/StepCreateBudgetLines.jsx

```diff
@@ -98,6 +115,60 @@ export const StepCreateBudgetLines = ({ goBack, goToNext, wizardSteps }) => {
         dispatch(setSelectedProject(-1));
     };
 
+    const handleCancelEdit = () => {
+        dispatch(setEditBudgetLineAdded({}));
+    };
+
+    const handleEditForm = (e) => {
+        e.preventDefault();
+        dispatch(
+            setEditBudgetLineAdded({
+                id: budgetLinesAdded[budgetLineBeingEdited].id,
+                line_description: enteredDescription,
+                comments: enteredComments,
+                can_id: selectedCan?.id,
+                can: selectedCan,
+                agreement_id: selectedAgreement?.id,
+                amount: enteredAmount,
+                date_needed:
+                    enteredYear && enteredMonth && enteredDay ? `${enteredYear}-${enteredMonth}-${enteredDay}` : null,
+                psc_fee_amount: selectedProcurementShop?.fee,
+            })
+        );
+        showAlert("success", "Budget Line Updated", "The budget line has been successfully edited.");
+    };
+
+    const handleSubmitForm = (e) => {
+        e.preventDefault();
+        dispatch(
+            setBudgetLineAdded([
+                ...budgetLinesAdded,
+                {
+                    id: crypto.getRandomValues(new Uint32Array(1))[0],
+                    line_description: enteredDescription || "",
+                    comments: enteredComments || "No comments",
```

Comment:
Is this intentional?  It saves "No comments" into the database

Reply:
Yeah, default to empty string